ArchiSteamFarm/Web/WebBrowser.cs

```diff
@@ -682,20 +682,6 @@ public HttpClient GenerateDisposableHttpClient(bool extendedTimeout = false) {
 		return null;
 	}
 
-	internal static void Init() {
-		// Set max connection limit from default of 2 to desired value
-		ServicePointManager.DefaultConnectionLimit = MaxConnections;
-
-		// Set max idle time from default of 100 seconds (100 * 1000) to desired value
-		ServicePointManager.MaxServicePointIdleTime = MaxIdleTime * 1000;
-
-		// Don't use Expect100Continue, we're sure about our POSTs, save some TCP packets
-		ServicePointManager.Expect100Continue = false;
-
-		// Reuse ports if possible
-		ServicePointManager.ReusePort = true;
```

Comment:
From what I can find at least some of these properties can be migrated to `HttpClient`, e.g. `ServicePointManager.Expect100Continue` => `HttpClient.DefaultRequestHeaders.ExpectContinue`. Worth checking which ones are still needed and can be migrated

Reply:
First two are migrated in sockets http handler, for expect 100 continue I need to check the default value, and for last one I didn't find replacement.

Reply:
It seems expect 100 continue defaults to not being used when not set, so we can skip it. Two other ones are already handled, and if you find replacement for ports reuse then let me know, until then I don't think we're done here :+1: 

![obraz](https://github.com/user-attachments/assets/97c680bf-86d4-464d-b2e7-9117f9b77ded)

Reply:
Can't find anything for `ReusePort` so we're good here

Reply:
For reference reusing ports probably has very limited usability anyway in our case, since the sockets http handler uses pooling of connections under the hood, so even if it exists and makes some difference, it's negligible and entirely not observable. The original service point manager option was added back in .NET Framework days.

Reply:
According to https://learn.microsoft.com/en-us/dotnet/api/system.net.servicepointmanager.reuseport?view=net-8.0 it might not even be used in http client at all, since it considers http web request :shrug: